client/components/domains/domain-transfer-suggestion/index.jsx

```diff
@@ -38,7 +38,7 @@ class DomainTransferSuggestion extends React.Component {
 						} ) }
 					</h3>
 					<p>
-						{ translate( "Transfer or map it to use it as your site's address.", {
+						{ translate( "You can use it as your site's address.", {
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Map this domain to use it as your site's address.', { context: 'Upgrades: Register domain description'} )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/domain-transfer-copy-styles)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?